Tasks/KubernetesV1/task.json

```diff
@@ -0,0 +1,404 @@
+{
+    "id": "CBC316A2-586F-4DEF-BE79-488A1F503564",
+    "name": "Kubernetes",
+    "friendlyName": "Deploy to Kubernetes",
+    "description": "Deploy, configure, update your Kubernetes cluster in Azure Container Service by running kubectl commands.",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=851275)",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "demands": [],
+    "releaseNotes": "What's new in Version 1.0: <br/>&nbsp;&nbsp; Use Azure service connection to select targeted AKS cluster.<br/>&nbsp;&nbsp;Output section has been removed as there already is an Output variables section.",
```

Comment:
Reword "What's new in Version 1.0: <br/>&nbsp;&nbsp; Use Azure service connection to select targeted AKS cluster.<br/>&nbsp;&nbsp;Output section has been removed as there already is an Output variables section." to 


What's new in Version 1.0: <br/>&nbsp;&nbsp; Added new connection type input for easy selection of Azure AKS cluster.<br/>&nbsp;&nbsp;Replaced output variable input with output variables section that we had added in all tasks

""

Reply:
Fix the formatting